pwem/viewers/viewer_localres.py

```diff
@@ -354,14 +353,13 @@ def getValuesRange(self):
             attrname = self.getEnumText('attrName')
             cifDic = AtomicStructHandler().readLowLevel(self.getAtomStructObject().getFileName())
             names, values = np.array(cifDic[NAME]), np.array(cifDic[VALUE])
-            recipient = cifDic[RECIP][0]
             attrValues = values[names == attrname]
             attrValues = list(map(float, attrValues))
             return min(attrValues), max(attrValues)
 
     def getColors(self):
       low, high = self.getValuesRange()
-      stepColors = splitRange(high, low, splitNum=self.intervals.get())
+      stepColors = splitRange(low, high, splitNum=self.intervals.get())
```

Comment:
You sure about this?

Reply:
I tested it a reasonable number of times and with these changes the colors were fine for all the representations